docs/_book.yaml

```
@@ -73,6 +73,8 @@ upper_tabs:
 
       ####  GOOGLE HARDWARE ####
       - heading: "Google hardware"
+      - title: "Access and Authentication"
```

Sentence-case is the devsite recommendation: 

```suggestion
      - title: "Access and authentication"
```